.travis.yml

```diff
@@ -6,9 +6,8 @@ matrix:
       jdk: oraclejdk9
     - os: osx
 
-script: >-
-    ./config/travis/run-checks.sh &&
-    time travis_retry ./gradlew clean checkstyleMain checkstyleTest test functional
+script:
+  - ./config/travis/run-checks.sh && time travis_retry ./gradlew clean checkstyleMain checkstyleTest test functional
```

Comment:
is there a need to even make the changes here?

Reply:
```>-``` is harder to maintain than ```-```

Reply:
For documentation:
According to @ongspxm, ```>-``` is ```block scalar style``` in ```yaml``` vs ```-``` which is a ```flow scalar style```.
The ```block scalar style``` treats subsequent multi-lines as a single line.